gax/src/test/resources/com/google/api/gax/rpc/mtls/mtls_context_aware_metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+  "cert_provider_command": [
+    "cat",
```

Comment:
Why are we doing it this way? It makes this java code platform-dependent. If you need just read the .pem file you don't need to use `cat` command for it, it can be and shoudl be read using java means. This is one of the typical differences between java and for example python - java code should not depend on system-wide tools (like cat in this case).

Reply:
Because a typical `context_aware_metadata.json` looks like this, it has a command we need to run in order to print the client cert to stdout. Here we just replace the command with `cat` for testing purposes.